deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -132,8 +161,37 @@ export class SolutionsInfrastructureStack extends Stack {
             role: bootstrapRole
         })
 
+        let vpc;
+        if (props.createVPC) {
+            vpc = new Vpc(this, 'Vpc', {});
+        }
+        else {
+            const vpcIdParameter = new CfnParameter(this, 'VPCId', {
+                type: 'AWS::EC2::VPC::Id',
+                description: 'Specify the VPC id to place Migration resources into'
+            });
+            addParameterLabel(parameterLabels, vpcIdParameter, "VPC Id")
+
+            const availabilityZonesParameter = new CfnParameter(this, 'VPCAvailabilityZones', {
+                type: 'List<AWS::EC2::AvailabilityZone::Name>',
+                description: 'Availability Zone names in the selected VPC to use. Must be the same NUMBER and ORDER of the selected Private Subnet Ids.'
+            });
+            addParameterLabel(parameterLabels, availabilityZonesParameter, "Availability Zones")
+
+            const privateSubnetIdsParameter = new CfnParameter(this, 'VPCPrivateSubnetIds', {
+                type: 'List<AWS::EC2::Subnet::Id>',
+                description: 'Private Subnet IDs in the selected VPC to use. Please provide exactly 2 or 3 subnets EACH in their own AZ. The subnets must have routes to a NAT gateway.'
+            });
+            addParameterLabel(parameterLabels, privateSubnetIdsParameter, "Private Subnet Ids")
```

Comment:
Lets drop the word 'id', while technically correct the detail isn't useful.
```suggestion
            addParameterLabel(parameterLabels, privateSubnetIdsParameter, "Private Subnets")
```